.github/workflows/docker-build.yml

```
@@ -0,0 +1,73 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - '*'
+
+  # Run tests for any PRs.
+  pull_request:
+
+env:
+  # The image name
+  IMAGE_NAME: docker-robot-framework:ci
+
+jobs:
+  # Run tests.
+  # See also https://docs.docker.com/docker-hub/builds/automated-testing/
+  build:
+    runs-on: ubuntu-latest
+    # run all our tests on each browser
+    strategy:
+      matrix:
+        include:
+          - browser: "chrome"
+          - browser: "firefox"
+    env:
+      BROWSER: ${{ matrix.browser }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build image
+        run: docker build . --tag $IMAGE_NAME --file Dockerfile
```

To be consistent with the lines below:

```suggestion
        run:
          docker build . --tag $IMAGE_NAME --file Dockerfile
```